hw_2/src/main/java/com/epam/spring/homework2/validator/MyValidator.java

```diff
@@ -0,0 +1,6 @@
+package com.epam.spring.homework2.validator;
+
+public interface MyValidator {
```

Comment:
Since this interface basically means - "whoever will implement me should validate itself"
The Validatable name would better fit here, please rename it

for ex. in java we have a lot of such interfaces(Comparable, Serializable etc)

Reply:
fixed